src/Riok.Mapperly/Descriptors/MappingBodyBuilders/ObjectPropertyMappingBodyBuilder.cs

```diff
@@ -211,48 +224,44 @@ public static void BuildMappingBody(IPropertiesContainerBuilderContext<IProperty
         // access the source in a null save matter (via ?.) but no other special handling required.
         if (delegateMapping.SourceType.IsNullable() || delegateMapping.IsSynthetic && targetPropertyPath.Member.IsNullable())
         {
-            var propertyMapping = new PropertyMapping(
-                delegateMapping,
-                sourcePropertyPath,
-                true,
-                false);
+            var propertyMapping = new PropertyMapping(delegateMapping, sourcePropertyPath, true, false);
             ctx.AddPropertyAssignmentMapping(new PropertyAssignmentMapping(targetPropertyPath, propertyMapping));
             return;
         }
 
         // additional null condition check
         // (only map if source is not null, else may throw depending on settings)
-        ctx.AddNullDelegatePropertyAssignmentMapping(new PropertyAssignmentMapping(
-            targetPropertyPath,
-            new PropertyMapping(delegateMapping, sourcePropertyPath, false, true)));
+        ctx.AddNullDelegatePropertyAssignmentMapping(
+            new PropertyAssignmentMapping(targetPropertyPath, new PropertyMapping(delegateMapping, sourcePropertyPath, false, true))
+        );
     }
 
     private static bool TryAddExistingTargetMapping(
         IPropertiesContainerBuilderContext<IPropertyAssignmentTypeMapping> ctx,
         PropertyPath sourcePropertyPath,
-        PropertyPath targetPropertyPath)
+        PropertyPath targetPropertyPath
+    )
     {
         // if the property is readonly
         // and the target and source path is readable,
         // we try to create an existing target mapping
-        if (targetPropertyPath.Member.CanSet()
+        if (
```

Comment:
Is this type of indentation intentionally? I find it a little confusing to read, as most C# projects do not format "ifs" this way.